portal/src/main/webapp/WEB-INF/jsp/mutation/mutationMapperTemplates.html

```diff
@@ -351,69 +351,65 @@
 </script>
 
 <script type="text/template" id="mutation_table_mutation_type_template">
-	<span class='{{mutationTypeClass}}'>
-		<label>{{mutationTypeText}}</label>
-	</span>
+	<span class='{{mutationTypeClass}}'>{{mutationTypeText}}</span>
 </script>
 
 <script type="text/template" id="mutation_table_cna_template">
-	<label alt='{{cnaTip}}' class='simple-tip {{cnaClass}}'>{{cna}}</label>
+	<span alt='{{cnaTip}}' class='simple-tip {{cnaClass}}'>{{cna}}</span>
 </script>
 
 <script type="text/template" id="mutation_table_cosmic_template">
-	<label class='{{cosmicClass}}'>{{cosmicCount}}</label>
+	<span class='{{cosmicClass}}'>{{cosmicCount}}</span>
```

Comment:
@pieterlukasse: Assistive technologies will parse `<label>` tags as labels for input elements/form controls. All such inputs require a label tag for accessibility and Section 508 compliance, but I believe that these spans here are not actually labels for inputs. If that’s correct, removing the `<label>` tags actually improves accessibility.

The `alt=` attribute has no meaning for other elements than `<img>` (it’s just sometimes confused with `title=`), and in this file it’s only used as a custom attribute to be read by the qTip library. That’s no issue for accessibility. 👍